reloj-checador/app/views/layouts/application.html.erb

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
```

Comment:
WDYT about adding the fontawesome assets via webpack?

1. console:
```bash
yarn add @fortawesome/fontawesome-free
```

2. import it in the javascript/packs/application.js:
```javascript
import "@fortawesome/fontawesome-free/css/all"
```

3. Check if it works:
```html
<i class="far fa-address-book"></i>
<i class="fab fa-github"></i>
````
That’s it!😊

Reply:
I didn't know that, I'm going to try xd